django_select2/conf.py

```diff
@@ -48,5 +48,38 @@ class Select2Conf(AppConf):
     It has set `select2_` as a default value, which you can change if needed.
     """
 
+    JS_LIB_FILE = '//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/js/select2.min.js'
+    """
+    The URI for the Select2 JS file. By default this points to the Cloudflare CDN.
+
+    If you want to select the version of the JS library used, or want to serve it from
+    the local 'static' resources, add a line to your settings.py like so::
+
+        SELECT2_JS_LIB_FILE = 'mylocaljslibs/select2.min.js'
+    """
+    CSS_LIB_FILE = '//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css'
+    """
+    The URI for the Select2 CSS file. By default this points to the Cloudflare CDN.
+
+    If you want to select the version of the library used, or want to serve it from
+    the local 'static' resources, add a line to your settings.py like so::
+
+        SELECT2_CSS_LIB_FILE = 'assets/css/select2-4.0.1.css.js'
+    """
+    JS_MEDIA = None
```

Comment:
Can change to `MEDIA_JS`, no probs.

MEDIA_JS is set to the CDN by default (in fact there is a test `test_default_media_is_cloudflare_4_0_0_version` to verify that). 

I thought including the `LIB_FILE` style settings is more friendly and having to define the whole MEDIA object.